src/cdi_analytic/Analytic_EICoupling.hh

```diff
@@ -0,0 +1,93 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   cdi_analytic/Analytic_EICoupling.hh
+ * \author Mathew Cleveland
+ * \date   March 2019
+ * \brief  Analytic_EICoupling class definition.
+ * \note   Copyright (C) 2016-2019 Triad National Security, LLC.
+ *         All rights reserved. */
+//---------------------------------------------------------------------------//
+
+#ifndef __cdi_analytic_Analytic_EICoupling_hh__
+#define __cdi_analytic_Analytic_EICoupling_hh__
+
+#include "Analytic_Models.hh"
+#include "cdi/EICoupling.hh"
+#include <memory>
+
+namespace rtt_cdi_analytic {
+
+//===========================================================================//
+/*!
+ * \class Analytic_EICoupling
+ *
+ * \brief Derived rtt_cdi::EICoupling class for analytic electron-ion 
+ * coupling data.
+ *
+ * The Analytic_EICoupling class is a derived rtt_cdi::EICoupling class. It provides
+ * analytic electron-ion coupling data.  The specific analytic EICoupling model is
+ * derived from the rtt_cdi_analytic::Analytic_EICoupling_Model base class.  Several
+ * pre-built derived classes are provided in Analytic_Models.hh.
+ *
+ * Clients of this class can provide any analytic model class as long as it
+ * conforms to the rtt_cdi_analytic::Analytic_EICoupling_Model interface.
+ *
+ * See the member functions for details about the data types and units.
+ *
+ * \example cdi_analytic/test/tstAnalytic_EICoupling.cc
+ *
+ * Example usage of Analytic_EICoupling, Analytic_EICoupling_Model, and their incorporation
+ * into rtt_cdi::CDI.
+ */
+// revision history:
```

Comment:
Remove 'history' section (git does this for us).  Also check 80 cols max for comments.